lib/gen_registry.ex

```diff
@@ -117,6 +117,28 @@ defmodule GenRegistry do
     ETS.info(table, :size)
   end
 
+  @doc """
+  Return a sample entry from the registry.
```

Comment:
nit: It'd be nice to clarify it's an unspecified entry, and not really random

Reply:
How does the sentence here not do that?  It just say a sample entry and does not say random.  Sample and unspecified seem like synonyms to me.